addons/website_event/static/src/js/website_event.js

```diff
@@ -41,26 +45,29 @@ var EventRegistrationForm = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     on_click: function (ev) {
+        var self = this;
```

Comment:
Let's not use `var`, and avoid introducing new `self = this` in the code. This shouldn't be necessary anymore with arrow functions.

I see you introduced it to be used within  `.then(function (modal) {`. Just write `.then((modal) => {` instead.